dockstore-cli-integration-testing/pom.xml

```diff
@@ -45,7 +45,7 @@
         <dependency>
             <groupId>io.dockstore</groupId>
             <artifactId>dockstore-client</artifactId>
-            <version>1.9.1-SNAPSHOT</version>
+            <version> 1.11.0-alpha.0-SNAPSHOT</version>
```

Comment:
Looks like an extra space in here and above

Reply:
fixed